src/auth/components/RegisterForm/style.less

```diff
@@ -62,11 +68,14 @@
 
           .input-box {
             width: 80px;
+            
           }
 
           .ant-select-selection {
             border: none;
             height: 27px;
+            background-color: var(--theme-background);
+            color: var(--theme-text-on-background-1);
```

Comment:
is this needed? (see above comment)

Reply:
Required for ANT dropdown menus to match background color.